diffpy/snmf/stretchednmfapp.py

```diff
@@ -33,7 +33,7 @@ def main():
     if args.input_directory is None:
         args.input_directory = Path.cwd()
     grid, data_input = load_input_signals(args.input_directory)
-    lifed_data_input = lift_data(data_input, args.lift_factor)
-    variables = initialize_variables(lifed_data_input,args.number_of_components,data_type='pdf')
+    lifted_data_input = lift_data(data_input, args.lift_factor)
```

Comment:
please can we change `data_input` to `input_data` everywhere (also with lifted_data_input).  `data_input` sounds like a verb (a function name) but I find `input_data` clearer.